koku/masu/database/report_db_accessor_base.py

```diff
@@ -424,7 +424,10 @@ def _commit_and_vacuum(self, table, sql, start=None, end=None):
             transaction.savepoint_commit(KokuDBAccess._savepoints.pop())
         with connection.cursor() as cursor:
             cursor.db.set_schema(self.schema)
-            cursor.execute(sql)
+            if bind_params is not None:
+                cursor.execute(sql, bind_params)
+            else:
+                cursor.execute(sql)
```

Comment:
I don't think a conditional is needed here. This should be functionally equivalent and using a keyword arg means not needing to care about whether `bind_params` is None or not:

```suggestion
                cursor.execute(sql, bind_params=bind_params)
```

Reply:
change added